torch/nn/utils/parametrize.py

```diff
@@ -277,7 +276,7 @@ def forward(self) -> Tensor:
 
 
 def _inject_new_class(module: Module) -> None:
-    r"""Sets up a module to be parametrized.
+    r"""Set up a module to be parametrized.
```

Comment:
Why this change? Same for the equivalent one in right_inverse and many others below.

These sentences refer to the function at hand, so they should be in third person. This is how it's done in most of the docs.

Reply:
@lezcano this is the change needed by pydocstyle - we are working on standardizing docstrings for pydocstyle.

Reply:
Fair enough. I personally prefer docs as a description of the function, but I guess that as long as it's consistent, the rest is less important.

Reply:
This addresses the `D401: First line should be in imperative mood` [pydocstyle rule](https://www.pydocstyle.org/en/stable/error_codes.html) which is in the list of fixes we were asked to address in the docathon issue.

```
torch/nn/utils/parametrize.py:280 in private function `_inject_new_class`:
        D401: First line should be in imperative mood (perhaps 'Set', not 'Sets')
```